Source/Tests/CMakeLists.txt

```diff
@@ -20,91 +20,28 @@
 # THE SOFTWARE.
 #
 
-# Define target name
-set (TARGET_NAME Urho3D)
+if (NOT ((APPLE AND NOT ARM) OR WIN32))
+    return ()
+endif ()
 
-if ((APPLE AND NOT ARM) OR WIN32)
-    
-    set (CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMake/Modules)
-    include (FindUrho3D)
-    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
-    # Add tests framework
-    # https://github.com/google/googletest/blob/master/googletest/README.md
-    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
+set (TARGET_NAME Tests)
 
-    # Download and unpack googletest at configure time
-    configure_file(CMakeLists.txt.in googletest-download/CMakeLists.txt)
-    execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
-        RESULT_VARIABLE result
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googletest-download )
+# Setup GTest
+include(FetchContent)
+FetchContent_Declare(
+    googletest
+    URL https://github.com/google/googletest/archive/e2239ee6043f73722e7aa812a459f54a28552929.zip
```

Comment:
This is totally opposite of what we normally do. This should end up in ThirdParty if we end up using it.